### PR TITLE
fix: Fixed undefined console errors faced while using with updated HFB

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
 - The issues arise when we changed the Registering of control from PHP to JS.

### Types of changes
Bugfix (non-breaking change which fixes an issue)

### How has this been tested?
 - Issues fixed.( Mostly undefined console errors )

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/custom-controls/class-astra-customizer-control-base.php [hunk 1]
```diff
@@ -74,6 +74,29 @@ public function enqueue_scripts() {
...
+					'600italic' => __( '600 Italic', 'astra' ),
+					'700'       => __( 'Bold 700', 'astra' ),
+					'700italic' => __( '700 Italic', 'astra' ),
+					'800'       => __( 'Extra-Bold 800', 'astra' ),
+					'800italic' => __( '800 Italic', 'astra' ),
+					'900'       => __( 'Ultra-Bold 900', 'astra' ),
+					'900italic' => __( '900 Italic', 'astra' ),
+				);
+
+				wp_localize_script( 'custom-control-script', 'astraTypo', $astra_typo_localize );
```
Comment (premanshup) [inline inc/customizer/custom-controls/class-astra-customizer-control-base.php:99]: @RajkiranBagal Please update the handle from `custom-control-script` to `astra-custom-control-script`. We released an update for this in the last week.

File: inc/customizer/custom-controls/class-astra-customizer-control-base.php [hunk 2]
```diff
@@ -103,6 +126,32 @@ public function enqueue_scripts() {
...
+					'600italic' => __( '600 Italic', 'astra' ),
+					'700'       => __( 'Bold 700', 'astra' ),
+					'700italic' => __( '700 Italic', 'astra' ),
+					'800'       => __( 'Extra-Bold 800', 'astra' ),
+					'800italic' => __( '800 Italic', 'astra' ),
+					'900'       => __( 'Ultra-Bold 900', 'astra' ),
+					'900italic' => __( '900 Italic', 'astra' ),
+				);
+
+				wp_localize_script( 'custom-control-react-script', 'astraTypo', $astra_typo_localize );
```
Comment (premanshup) [inline inc/customizer/custom-controls/class-astra-customizer-control-base.php:154]: @RajkiranBagal Please update the handle from `custom-control-react-script` to `astra-custom-control-react-script`. We released an update for this in the last week. Verify once if the localized variables are working properly or not.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/customizer/custom-controls/class-astra-customizer-control-base.php (hunk 2 above)
Comment (RajkiranBagal) [reply inc/customizer/custom-controls/class-astra-customizer-control-base.php]: Done

File: inc/customizer/custom-controls/class-astra-customizer-control-base.php (hunk 1 above)
Comment (RajkiranBagal) [reply inc/customizer/custom-controls/class-astra-customizer-control-base.php]: Done

**Review by premanshup — APPROVED** [review]

**Pull request merged**